### ISSUE TITLE
the language is undefined

### ISSUE THREAD

**Issue by zzzgit** [opr/WiktionarySearch#4]
`https://undefined.wiktionary.org/wiki/keyword`
this page will be opened after I performed this plugin.

**Comment by opr** [comment]
I'll investigate this, for now the fix is: go to chrome://extensions and
open the options of WiktionarySearch, then change the language to something
different then back to the language you want to use

**Comment by zzzgit** [comment]
yes thanks i see you have updated this plugin in 2017, but the problem remains

**Comment by HuskyNator** [comment]
I have the same issue sadly.
Note I surmise this happens because the initial language it used was different & I had to set it to english under the settings (causing this issue).

Strangest thing is: after switching to afrikaans, aenglisc & back to english again, the issue resolved. 🤷‍♀️
